drivers/espi/espi_it8xxx2.c

```diff
@@ -209,7 +209,7 @@ static const struct ec2i_t pmc2_settings[] = {
  */
 #define IT8XXX2_ESPI_H2RAM_POOL_SIZE_MAX 0x1000
 #define IT8XXX2_ESPI_H2RAM_OFFSET_MASK   GENMASK(5, 0)
-#define IT8XXX2_ESPI_H2RAM_BASEADDR_MASK (KB(CONFIG_SRAM_SIZE) - 1)
+#define IT8XXX2_ESPI_H2RAM_BASEADDR_MASK 0xfffff
```

Comment:
nit: use GEMASK macro to create contiguous bitmask.

Reply:
Done.